src/main/java/core/basesyntax/EmployeeService.java

```diff
@@ -1,10 +1,25 @@
 package core.basesyntax;
 
+import java.util.Comparator;
 import java.util.List;
 import java.util.Set;
+import java.util.TreeSet;
 
 public class EmployeeService {
     public Set<Employee> getEmployByOrder(List<Employee> employees) {
-        return null;
+        Comparator<Employee> comparator = new Comparator<Employee>() {
+            @Override
+            public int compare(Employee firstEmployee, Employee secondEmployee) {
+                int compare = Integer.compare(firstEmployee.getAge(), secondEmployee.getAge());
```

Comment:
let's make the variable name more informative

Reply:
done